### PR TITLE
Serverless specificaction with v08 spec

### PR DESCRIPTION
**Pull request by spolti**
This PR aims to align the sdk with the v08 spec.
Fixes: https://github.com/serverlessworkflow/sdk-go/issues/70

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
Initial alignment with v08 spec

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,6 +19,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/leodido/go-urn v1.2.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/serverlessworkflow/sdk-go v1.0.0 // indirect
```
Comment (ricardozanini) [inline go.mod:22]: This is the old version of the SDK. Why it was added here? Can you review the imports?

Comment (spolti) [reply go.mod:22]: I honestly don't know why. probably something with my ide.

**Comment by ricardozanini** [comment]
I think we should map all the required changes for v0.8 and handle them in this PR or strict the scope of this one. "v08spec" as a title seems too wide.

**Comment by lsytj0413** [comment]
> I think we should map all the required changes for v0.8 and handle them in this PR or strict the scope of this one. "v08spec" as a title seems too wide.

Maybe make a PR do one change, and mark all of them as v0.8 Milestone？

**Comment by spolti** [comment]
> e should map all the required changes for v0.8

the changes needed are mapped here https://github.com/serverlessworkflow/sdk-go/issues/70
Or are you suggesting something different?

**Comment by ricardozanini** [comment]
@spolti open PRs to change to 0.8 like what @lsytj0413 is doing. So we can keep them on track using the milestone feature.

**Comment by ricardozanini** [comment]
Or if this is a small change (which I'm not sure of), you can do in this one PR.

**Comment by spolti** [comment]
ok...
let me break each commit in different prs...

**Comment by spolti** [comment]
suppressed by:
- https://github.com/serverlessworkflow/sdk-go/pull/80

will send the other PRs as soon as the previous gets merged as it depends on each other.

**Pull request closed without merging**